File: src/main/java/com/rs/java/game/area/multi/AlkharidMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.shapes.Rectangle;

public final class AlkharidMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Alkharid Multi";
	}

	public AlkharidMulti() {
		super(13105);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/AlkharidMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class AlkharidMulti : Area(13105) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Alkharid Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
